scripts/zkai_d64_external_recursion_adapter_gate.py

```
@@ -0,0 +1,816 @@
+def mutated_receipts() -> dict[str, tuple[str, dict[str, Any]]]:
+    baseline = baseline_receipt()
+
+    def mutate(name: str, surface: str, fn: Callable[[dict[str, Any]], None], *, refresh: bool = True) -> None:
+        receipt = copy.deepcopy(baseline)
+        fn(receipt)
+        if refresh:
+            _refresh_statement_commitment(receipt)
+        out[name] = (surface, receipt)
+
+    out: dict[str, tuple[str, dict[str, Any]]] = {}
+    nested = lambda r: r["statement"]["source_contract"]["nested_verifier_contract"]
```

Ruff (0.15.12)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `nested` as a `def`

(E731)